src/sentry/api/base.py

```diff
@@ -160,6 +161,27 @@ def load_json_body(self, request):
         except json.JSONDecodeError:
             return
 
+    def _create_api_access_log(self):
+        """
+        Create a log entry to be used for api metrics gathering
+        """
+        try:
+            token_class = getattr(self.request.auth, "__class__", None)
+            token_name = token_class.__name__
+            view_obj = self.request.parser_context["view"]
+            log_metrics = dict(
+                method=self.request.method,
+                view=f"{view_obj.__module__}.{view_obj.__class__.__name__}",
+                response=self.response.status_code,
+                user=str(self.request.user),
+                token_type=token_name,
+            )
+            api_access_logger.info(log_metrics)
+        except Exception as exc:
+            api_access_logger.error(
```

Comment:
Let's use `.exception` here so that we capture the actual exception too. Then you don't need to include `exc` in your string format.